.changeset/fifty-words-lick.md

```
@@ -0,0 +1,5 @@
+---
+"superfill.ai": patch
+---
+
+add obersvability in the document parser
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)